src/components/TabbedNav/TabbedNav.jsx

```diff
@@ -0,0 +1,42 @@
+const React = require('react')
+const hookUpStickyNav = require('../hookUpStickyNav')
+
+class TabbedNav extends React.Component {
+  render() {
+    return (
+      <nav id="tabbed-nav" className="top-nav nav tabs is-toggle is-fullwidth">
+        <ul>
+          <li className={this.props.active === 'tasks' ? 'is-active' : ''}>
```

Comment:
Could do `const { active } = this.props` just above the `return` on line 6, to shorten these lines.